docs/reference/cluster-config.txt

```diff
@@ -294,6 +296,18 @@ The master supports the following configuration settings:
       -  ``serveraddress`` (optional)
       -  ``email`` (optional)
 
+   -  ``add_capabilities``: The default list of capability strings to
+      pass to the Docker daemon. Ignored by resource managers of type
```

Comment:
Maybe similar rewording elsewhere
```suggestion
   -  ``add_capabilities``: The default list of Linux capabilities to
      grant to task containers. Ignored by resource managers of type
```

Reply:
done; affected experiment config, cluster config, and command/notebook config docs.